app/controllers/api/v1/lead_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::LeadController < ApplicationController
+    before_action :authorize_request, except: [:create]
+    before_action :get_authorize_user, only: [:create]
+
+    def create    
+        unless @current_user
```

Comment:
What happen with the users that already are logged in? how will they create the lead?

Reply:
Solved